lib/foreman_chef/engine.rb

```diff
@@ -33,6 +33,10 @@ class Engine < ::Rails::Engine
       ActionView::Base.send :include, ChefProxyForm
     end
 
+    initializer 'foreman_chef.dynflow_world' do |app|
+       ForemanTasks.dynflow.require!
```

Comment:
In foreman_salt, we made it optional, kinda hacky but it works: https://github.com/theforeman/foreman_salt/blob/master/lib/foreman_salt/engine.rb#L13

But, it is quickly coming to be used in core - so I think we should get it packaged for Debian sooner rather than later.